packages/suite/src/views/wallet/coinmarket/common/CoinmarketIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+import { iconSizes, Image, Row } from '@trezor/components';
+import { borders } from '@trezor/theme';
+
+import { useSelector } from 'src/hooks/suite';
+
+const CoinmarketIconWrapper = styled.div<{ $isDark: boolean }>`
+    ${({ $isDark }) => $isDark && `background-color: #fff;`}
+    border-radius: ${borders.radii.xxs};
+`;
+
+interface CoinmarketIconProps {
+    iconUrl: string;
+}
+
+export const CoinmarketIcon = ({ iconUrl }: CoinmarketIconProps) => {
+    const currentTheme = useSelector(state => state.suite.settings.theme.variant);
+    const iconWrapperSize = iconSizes.mediumLarge + 4;
```

Comment:
what is this magical number 4? :D Maybe it's easier to set iconWrapperSize manually like: 

```
const ICON_WRAPPER_SIZE = 123
```

Reply:
It is total padding in the wrapper. (2px left - `mediumLarge` size of image - 2px right)

Reply:
OK, then I suggest to use it this way:

```ts
const iconWrapperSize = iconSizes.mediumLarge + 2 * SIDE_PADDING;
```